polaris-core/src/main/java/org/apache/polaris/core/entity/CatalogEntity.java

```diff
@@ -77,7 +78,7 @@ public static CatalogEntity of(PolarisBaseEntity sourceEntity) {
     return null;
   }
 
-  public static CatalogEntity fromCatalog(Catalog catalog) {
+  public static CatalogEntity fromCatalog(Catalog catalog, PolarisCallContext callContext) {
```

Comment:
PolarisCallContext is typically the first argument in the list. I'd maintain that pattern